### ISSUE TITLE
TRANSLATION_GUIDE 반영 번역어 통일

### ISSUE THREAD

**Issue by rltq** [PyTorchKR/tutorials-kr#1133]
Related : #1063

## 문서 URL
수정 문서가 10여건 이상으로 페이지가 아닌 단어를 중심으로 작성합니다.

## 변경 사항
특정 용어들이 [TRANSLATION_GUIDE](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)와 다르게 번역되는 사례를 발견했고, 그렇게 발견 된 단어들을 (수기) 전수조사하여 정정했습니다. 

### 1. 음차 표기 사례를 가이드 라인에 맞게 정정

> 음차 표기되어 의미 이해에 영향이 없을 수 있으나 혼용되어 표기된 사례가 상당해 통일이 필요한 항목입니다.

#### '레이어'를 '계층'으로 변경 
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L87

혼용 사례가 가장 많습니다. 직역도 의미는 통하나 가이드에 따르면 원칙적으로 '계층'으로 통일되어야 합니다.

- advanced_source/neural_style_tutorial.py
- advanced_source/semi_structured_sparse.py
- beginner_source/basics/optimization_tutorial.py
- beginner_source/blitz/neural_networks_tutorial.py
- beginner_source/introyt/introyt1_tutorial.py
- beginner_source/introyt/tensorboardyt_tutorial.py
- beginner_source/nlp/sequence_models_tutorial.py
- beginner_source/chatbot_tutorial.py
- beginner_source/dcgan_faces_tutorial.py
- beginner_source/fgsm_tutorial.py
- beginner_source/finetuning_torchvision_models_tutorial.py
- beginner_source/nn_tutorial.py
- beginner_source/torchtext_translation.py
- beginner_source/transformer_tutorial.py
- intermediate_source/char_rnn_classification_tutorial.py
- intermediate_source/char_rnn_generation_tutorial.py
- intermediate_source/pipeline_tutorial.py
- intermediate_source/seq2seq_translation_tutorial.py
- recipes_source/recipes/tuning_guide.py
- recipes_source/recipes/warmstarting_model_using_parameters_from_a_different_model.py

#### '파라미터'를 '매개변수'로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L104

역시 혼용되는 사례가 많으나 가이드에 따르면 '매개변수'로 번역되는 것이 옳습니다.

- advanced_source/static_quantization_tutorial.rst
- beginner_source/basics/optimization_tutorial.py
- beginner_source/introyt/introyt1_tutorial.py
- beginner_source/nlp/deep_learning_tutorial.py
- beginner_source/fgsm_tutorial.py
- beginner_source/finetuning_torchvision_models_tutorial.py
- beginner_source/hyperparameter_tuning_tutorial.py
- intermediate_source/ensembling.py
- intermediate_source/pipeline_tutorial.py
- intermediate_source/pruning_tutorial.py
- intermediate_source/spatial_transformer_tutorial.py
- intermediate_source/TP_tutorial.rst
- recipes_source/recipes/amp_recipe.py
- recipes_source/recipes/tuning_guide.py
- recipes_source/recipes/zeroing_out_gradients.py
- recipes_source/compiling_optimizer.rst

### 2. 번역어 혼용 사례를 가이드라인에 맞게 정정

> 일상적으로 혼용되어 사용되고 있고, 의미 이해에 영향을 줄 수 있는 항목들입니다.

#### 3. '인수'를 '인자'로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L50

- beginner_source/introyt/tensorboardyt_tutorial.py
- beginner_source/nlp/deep_learning_tutorial.py
- beginner_source/nlp/sequence_models_tutorial.py
- intermediate_source/tensorboard_profiler_tutorial.py
- recipes_source/recipes/amp_recipe.py
- recipes_source/recipes/tuning_guide.py
- recipes_source/torch_compile_user_defined_triton_kernel_tutorial.py

#### '오차 행렬'을 '혼동 행렬'로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L60

- intermediate_source/tiatoolbox_tutorial.rst

#### '기울기'를 '변화도'로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L77

- advanced_source/neural_style_tutorial.py
- advanced_source/numpy_extensions_tutorial.py
- beginner_source/introyt/introyt1_tutorial.py
- beginner_source/nlp/sequence_models_tutorial.py
- beginner_source/nlp/word_embeddings_tutorial.py
- beginner_source/nn_tutorial.py
- beginner_source/transformer_tutorial.py
- intermediate_source/autograd_saved_tensors_hooks_tutorial.py
- intermediate_source/forward_ad_usage.py
- intermediate_source/optimizer_step_in_backward_tutorial.py
- intermediate_source/pipeline_tutorial.py
- intermediate_source/tiatoolbox_tutorial.rst

#### '훈련'을 '학습'으로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L124

PR 검토 중 diff 확인 과정에서 training이 훈련으로 직역되고 있음을 확인했습니다. 정정이 필요한 페이지 수가 다소 많게 느껴지나 혼용된 사례 유형은 일관됩니다. 용어집과 일반적인 용례를 고려할 때 '훈련'으로 사용 된 단어를 '학습'으로 통일했습니다.

레이어를 계층으로 통일한 항목과 다르게 본 항목은 페이지 하나당 한두개정도의 정정만 필요한 상황입니다.

- advanced_source/neural_style_tutorial.py
- beginner_source/basics/autogradqs_tutorial.py
- beginner_source/introyt/introyt1_tutorial.py
- beginner_source/introyt/tensorboardyt_tutorial.py
- beginner_source/nlp/advanced_tutorial.py
- beginner_source/nlp/deep_learning_tutorial.py
- beginner_source/nlp/sequence_models_tutorial.py
- beginner_source/nlp/word_embeddings_tutorial.py
- beginner_source/dcgan_faces_tutorial.py
- beginner_source/fgsm_tutorial.py
- beginner_source/finetuning_torchvision_models_tutorial.py
- beginner_source/hyperparameter_tuning_tutorial.py
- beginner_source/nn_tutorial.py
- beginner_source/torchtext_translation.py
- beginner_source/transformer_tutorial.py
- intermediate_source/autograd_saved_tensors_hooks_tutorial.py
- intermediate_source/char_rnn_classification_tutorial.py
- intermediate_source/forced_alignment_with_torchaudio_tutorial.py
- intermediate_source/mario_rl_tutorial.py
- intermediate_source/pipeline_tutorial.py
- intermediate_source/reinforcement_q_learning.py
- recipes_source/recipes/amp_recipe.py
- recipes_source/recipes/custom_dataset_transforms_loader.py
- recipes_source/recipes/defining_a_neural_network.py
- recipes_source/recipes/tuning_guide.py
- recipes_source/recipes/warmstarting_model_using_parameters_from_a_different_model.py
- recipes_source/recipes/zeroing_out_gradients.py

### 3. 표기 또는 일관되게 발견되는 오탈자 정정

> 의미 이해에 어려움은 없으나 원칙상 정정되어야 하는 항목입니다.

#### 'Dropout', '드랍아웃' 등을 '드롭아웃'으로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L62

- beginner_source/chatbot_tutorial.py
- beginner_source/transformer_tutorial.py
- intermediate_source/pipeline_tutorial.py

#### '메서드'를 '메소드'로 변경
https://github.com/PyTorchKorea/tutorials-kr/blob/99b4e260e699798b31c7a9a9517bc8125aab2b5b/TRANSLATION_GUIDE.md?plain=1#L94

- advanced_source/semi_structured_sparse.py
- beginner_source/blitz/neural_networks_tutorial.py
- beginner_source/introyt/introyt1_tutorial.py
- beginner_source/nlp/deep_learning_tutorial.py
- beginner_source/chatbot_tutorial.py
- intermediate_source/pipeline_tutorial.py
- recipes_source/recipes/timer_quick_start.py

## 추가 정보
- [TRANSLATION_GUIDE](https://github.com/PyTorchKorea/tutorials-kr/blob/master/TRANSLATION_GUIDE.md)
- 관련 PR
